### ISSUE TITLE
[GH Request] Onboard 2U employees David Nuon, Earnest Berry, German Olle, and Marcos Rigoli

### ISSUE THREAD

**Issue by ashultz0** [openedx/axim-engineering#747]
### Firm Name

2U

### User List (case sensitive!) - Please also include in ticket title

Employee Full Name
githubhandle

David Nuon
davidnuon
Earnest Berry
Souvent22
German Olle
germanolleunlp
Marcos Rigoli
rijuma

### Start Date

17 May 2023

### Urgency

Medium (< 2 weeks)

**Comment by e0d** [comment]
@ashultz0 which team will these folks be joining?  We're trying to roll out more fine-grained teams than push-pull-all whenever possible.

**Comment by e0d** [comment]
Can we also get emails for our records.

**Comment by e0d** [comment]
They have been added to the CRM and will be in the CLA check file this afternoon.

**Comment by ashultz0** [comment]
it's a new team (well it's an older 2U team but new to edx use) and work is going to be in lms/studio/frontends for same so I'm not sure if a team will help a lot. The team name in 2U is DLX "Digital Learning Experience" which doesn't actually mean anything.

David Nuon / dnuon@2u.com / davidnuon
Earnest Berry / eberry@2u.com / Souvent22
German Olle / golle@2u.com / germanolleunlp
Marcos Rigoli / mrigoli@2u.com / rijuma

**Comment by e0d** [comment]
I've created a team and added:

* edx-platform
* frontend-app-account
* frontend-app-learner-portal-programs
* frontend-app-learning
* frontend-app-library-authoring
* studio-frontend

If there are other requirements please let me know.

**Comment by ashultz0** [comment]
@e0d what's the team? I found https://github.com/orgs/openedx/teams/2u-digital-learning-experience which seems right but none of us are on it yet.

**Comment by e0d** [comment]
@ashultz0 all the above folks will have received an invite that needs to be accepted to join.  Shall I add you to the team as well?

**Comment by ashultz0** [comment]
yes please! Will we have to come back to get future devs on or will members be able to pull in new members?

**Comment by feanil** [comment]
@ashultz0 you'll have to make a new request to get more people added.

**Comment by ashultz0** [comment]
https://openedx.atlassian.net/wiki/spaces/COMM/pages/3771793429/Memo+Granting+write+access+to+openedx+repos looks like we should still be adding 2U employees to push-pull-all, is that memo or the action on this PR the latest decision?

**Comment by e0d** [comment]
I am confident that we will be migrating away from push-pull-all over time.  If you are convinced that you need it now, we can add the team members to it.  The fact is that users do not commit to all the repos in push-pull-all, so the access is overkill.  The tangible difference is need to use the fork workflow, which Ned documents [here](https://openedx.atlassian.net/wiki/spaces/AC/pages/3458170881/Working+with+Personal+Forks).

If this is causing you friction or hardship, I'll add the folks.  

Do note that new repositories aren't being added to push-pull-all, so it's a misnomer at this point.
